src/Psalm/Type/Atomic/TIntRange.php

```diff
@@ -70,4 +70,12 @@ public function isPositive(): bool
     {
         return $this->min_bound !== null && $this->min_bound > 0;
     }
+
+    public function contains(int $i): bool
+    {
+        return
+            ($this->min_bound === null && $this->max_bound === null) ||
+            ($this->min_bound === null && $this->max_bound >= $i) ||
+            ($this->max_bound === null && $this->min_bound <= $i);
```

Comment:
Shouldn't there be another case for when both min and max are specified?

Reply:
Thanks!